formatters/svg/svg.go

```diff
@@ -38,12 +33,8 @@ func EmbedFontFile(fontFamily string, fileName string) (option Option, err error
 		return nil, errors.New("unexpected font file suffix")
 	}
 
-	if f, err := os.Open(fileName); err == nil {
-		defer f.Close()
-		reader := bufio.NewReader(f)
-		if content, err := ioutil.ReadAll(reader); err == nil {
-			option = EmbedFont(fontFamily, base64.StdEncoding.EncodeToString(content), format)
-		}
+	if content, err := ioutil.ReadFile(fileName); err == nil {
```

Comment:
Do we want to return an error here if we can't load the font?

Reply:
seems i messed up the named return with `:=`. Fixed it with `:`, but non named return with if/else might be an alternative as well.

Reply:
error message itself looks fine to me: `open nonexistent.woff: no such file or directory`